src/Freya.Core/Integration.fs

```diff
@@ -18,12 +18,12 @@
 //
 //----------------------------------------------------------------------------
 
-[<AutoOpen>]
-module Freya.Core.Integration
+module Freya.Integration
```

Comment:
Sounds good to me. I was trying to make minimal changes. `Pipeline` and `Integration` are slightly different from the `Core` module types, so I don't think it's bad to leave them, but I also can't argue against using the same module/namespace for all.

Are you thinking `Freya.Core.Pipeline` and `Freya.Core.Integration` or just `Freya.Core` for everything?